### PR TITLE
Catalan translation

### PR DESCRIPTION
**Pull request by ThiEstoRil**
# SCP:SL Translation PR Template

## Language
Català (Catalan)

## Type of change
- [x] Added new language translation
- [ ] Changed a currently existing translation

## Have you tested the changes you've made?
- [X] Yes
- [ ] No

# Checklist:
- [X] I have made sure to not commit changes to other stuff than my own or the default english one
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [X] I have made sure that I'm not submitting a duplicate translation 
- [X] I have not included any advertisements/pointers to non-game stuff other than author in the folder name
- [X] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of new translations

### CODE REVIEW

File: Translations/credits.txt [hunk 1]
```diff
@@ -25,5 +25,6 @@ sk by Gejb & Mordi Slovakian
 th by AGKK Histoire & DatPanda
 tr by Jhrino, MHS, vexdy, Gonsez
 uk by TRI99ER
-zh-Hans by Schrodinger_Sig, ShingekiNoRex, TitanSix & Tychus_Xiaoyi
-zh-Hant by X-EROS
\ No newline at end of file
+zh-Hans by Schrodinger_Sig & ShingekiNoRex
+zh-Hant by X-EROS
+ca by ThiEstoRil
```
Comment (Rnen) [inline Translations/credits.txt]: Please do not remove members from another translation.
Your submission wil be on hold untill this is resolved.
Also, languages are to be sorted alphabetically.

Comment (ThiEstoRil) [reply Translations/credits.txt]: Done! The fork I made is a few months ago, so I updated the list of credits.
I have reused "EncodingNormalizer.exe" to the new / modified files.
I have also sorted alphabetically the list of credits.

**Comment by Rnen** [comment]
# Missing Lines
One of our moderators has noticed the translation is missing lines.
Please check your translation for anything missing.
We have just added a tool that does this automatically, which might make this process easier.

---
https://github.com/northwood-studios/SCPSL-Translations/blob/master/TranslationChecker.exe

**Comment by ThiEstoRil** [comment]
> # Missing Lines
> One of our moderators has noticed the translation is missing lines.
> Please check your translation for anything missing.
> We have just added a tool that does this automatically, which might make this process easier.
> 
> https://github.com/northwood-studios/SCPSL-Translations/blob/master/TranslationChecker.exe

Okay. All done!

File: Translations/ca/Facility.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+OPCIONS
+DESCONECTAR
+SITJA ALFA DESHABILITAT
+ACCÈS DENEGAT
+CHAT DE VEU
+INVENTARI SOBRECARREGAT
+M. SENSIBILITAT
+HACKEJANT
+RECOLLINT...
+Incrementar la brillantor <b>empitjora la jugabilitat </b>del joc.\nTracti d'ajustar-ho el m�nim possible.\n<b>�s recomanable deixar el nivell per defecte.</b>
```
Comment (MichalPetryka) [inline Translations/ca/Facility.txt]: Some characters got corrupted.

File: Translations/ca/Facility.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+[1] OBRIR/TANCAR PUERTA\n[2] BLOQUEJAR PORTA*\n\n*Bloquejar puertes gasta la habilitat especial
+LA PORTA HA SIGUT SELLADA PER L'SCP-079
+Pantalla Completa
+Segur que vols abandonar el servidor?
+Opcions bàsiques. Llista d'opciones completes en el men�.
```
Comment (MichalPetryka) [inline Translations/ca/Facility.txt]: Same here.

File: Translations/ca/Facility.txt (hunk 2 above)
Comment (ThiEstoRil) [reply Translations/ca/Facility.txt]: Fixed. Thanks!

**Comment by Rnen** [comment]
Thank you for your cooperation, it has been a pleasure :grin: 
The translation will now be merged :ok_hand:

**Pull request merged**